### ISSUE TITLE
Wraparound tabs stopped working after 3.0.0

### ISSUE THREAD

**Issue by wgolledge** [mohnish/rearrange-tabs#60]
First off, I love this extension - thanks for creating it @mohnish!

### Problem

Previously when a tab was at "the end", it could be "wrapped around" by pressing the move key again. E.g. If a tab is in the very right-most position and I press the _move selected tab(s) right_ hotkey the tab would move into the first position (far left).

This isn't major by any means but I really enjoyed moving the tabs I deemed to be the most important back to the front in this way before clearing out the rest of my tabs.

### Solution

- Tabs should move to the start/end if the move key is pressed while they're in the last/first position

### Notes

I'd be up for trying to implement this functionality if desired

**Comment by mohnish** [comment]
Hello @wgolledge 👋 

I'm glad you love the extension. It's users like you and the amazing contributions from the community we have here on GitHub that makes this extension work well.

Regarding the issue that you've brought up, that's the expected behavior now. I removed the wrap-around feature in 3.0.0. I mentioned about it here: https://iam.mt/rearrange-tabs-3-0-0/

A little history around that feature:

I introduced it early on (v1) when the option to move to front/back (v2) wasn't implemented. When I worked on the 3.0 rewrite, I carefully weighed in on which aspects of the extension seemed redundant, buggy, or causing perf issues, and this one fit the redundant category. I tried to clean up the extension and make it lean. I hope you're able to use the move back/front instead. The move to back/front is actually much more efficient too.

If you are interested more around this decision, here's another issue https://github.com/mohnish/rearrange-tabs/issues/55 where I've discussed this.

Hope the other shortcuts fit in to your use case. Cheers!
